modules/cnf-reducing-netqueues-using-pao.adoc

```diff
@@ -0,0 +1,349 @@
+
+. Create a performance profile that is appropriate for your hardware and topology. For more information see xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#cnf-tuning-nodes-for-low-latency-via-performanceprofile_cnf-master[Tuning nodes for low latency with the performance profile]. Alternatively, edit an existing performance profile using the following command:
++
+[source,terminal]
+----
+$ oc edit performanceprofile <your-profile-name>
+----
+
+. Populate the `spec` field profile with the `net` object. The object list may contain two fields:
+
+* `userLevelNetworking` (required) specifies a boolean flag which if true sets the queue count to the reserved CPU count for all supported devices. The default is false.
+* `Devices` (optional) specifies a list of devices that will have the queues set to the reserved CPU count. If the device list is empty the configuration applies to all network devices. The configuration is as follows:
+** `interfaceName`: Name supports shell-style wildcards which can be positive or negative.
+*** Example wildcard syntax is as follows: <string> .*
+*** Negative rules are prefixed with an exclamation mark. To apply the net queue changes to all devices other than the excluded list use  `!<device>` for example !eno1.
+* `vendorId`: Network device vendor ID represented as a 16 bit hexmadecimal number with a 0x prefix.
+* `deviceId`: Network device ID (model) represented as a 16 bit hexmadecimal number with a 0x prefix.
++
+[NOTE]
+====
+When a `deviceId` is specified the `vendorId` must also be defined.
+A device that matches all of the device identifiers (`interfaceName`, `vendorId` or a pair of `vendorId` plus `deviceId` representing a net device) qualifies as a network device that will have its net queues count set to the reserved CPU count.
+When two or more devices are specified then the net queues count is set to any net device that matches one of them.
+====
++
+An example performance profile configuration is shown below:
++
+[source,yaml]
+----
+  apiVersion: performance.openshift.io/v2
+  kind: PerformanceProfile
+  metadata:
+    name: manual
+  spec:
+    cpu:
+    isolated: 3-51,54-103
+    reserved: 0-2,52-54
+
+  #applied on all net devices found by tuned (device list is empty = all devices)
+    net:
+    - userLevelNetworking: true  <1>
+    #!
+
+    more examples:
+
+      net:
+        userLevelNetworking: true
+        devices:
+            -interfaceName: “eth0”
+            -interfaceName: “eth1”   <2>
+            -vendorId: “0x1af4”
+            deviceId: “0x1000”
+
+      net:
+       userLevelNetworking: true    <3>
+       devices:
+        -interfaceName: “eth*”
+
+      net:
+       userLevelNetworking: true    <4>
+       devices:
+        -interfaceName: “!eno1”
+
+     net:
+       userLevelNetworking: true    <5>
+       devices:
+        -interfaceName: “eth0”
+        -vendorId: “0x1af4”
```

Comment:
remove the `-` before vendorID and make sure all of the items align (having the same color as well)